cluster-provision/k8s/1.17/provision.sh

```diff
@@ -121,9 +121,49 @@ echo "net.netfilter.nf_conntrack_max=1000000" >> /etc/sysctl.conf
 
 systemctl restart NetworkManager
 
+mkdir -p /tmp/kubeadm-patches/
+
+cat >/tmp/kubeadm-patches/kustomization.yaml <<EOF
+patchesJson6902:
+- target:
+    version: v1
+    kind: Pod
+    name: kube-apiserver
+    namespace: kube-system
+  path: add-security-context.yaml
+- target:
+    version: v1
+    kind: Pod
+    name: kube-controller-manager
+    namespace: kube-system
+  path: add-security-context.yaml
+- target:
+    version: v1
+    kind: Pod
+    name: kube-scheduler
+    namespace: kube-system
+  path: add-security-context.yaml
+- target:
+    version: v1
+    kind: Pod
+    name: etcd
+    namespace: kube-system
+  path: add-security-context.yaml
+EOF
+
+cat >/tmp/kubeadm-patches/add-security-context.yaml <<EOF
+- op: add
+  path: /spec/containers/0/
+  value:
+    securityContext:
+      seLinuxOptions:
+        type: spc_t
```

Comment:
@slintes @rmohr maybe you guys have an idea what am I doing wrong here...
the end result of it this is just `securityContext: {}`
I've tried adding `-` to the path but nothing really worked.

Reply:
In a simple reproducer using `kustomize` directly it worked after removing the trailing `/` in `path: /spec/containers/0/`

Reply:
@slintes Thank you!